osu.Game/Users/Drawables/UpdateableAvatar.cs

```diff
@@ -69,14 +69,20 @@ protected override Drawable CreateDrawable(User user)
             if (user == null && !showGuestOnNull)
                 return null;
 
-            var avatar = new ClickableAvatar(user)
+            if (!openOnClick)
+            {
+                return new DrawableAvatar(user)
+                {
+                    RelativeSizeAxes = Axes.Both,
+                };
+            }
+
```

Comment:
Rewrite as something like

```suggestion
            if (!openOnClick)
            {
                return new DrawableAvatar(user)
                {
                    RelativeSizeAxes = Axes.Both,
                };
            }
            else
            {
            ...
```

Right now it reads very unbalanced.